src/graphql/resolvers/getByDOI.js

```diff
@@ -138,10 +138,26 @@ const getByDOI = {
       throw Error('Internal server error');
     }
 
+    console.log(apiKeyConfig);
+
     if (!Array.isArray(apiKeyConfig?.access) || !apiKeyConfig?.access?.includes('graphql') || !apiKeyConfig?.allowed) {
       throw Error('Not authorized');
     }
 
+    // Demo apikey
+    if (apikey === 'demo') {
+      if (apiKeyConfig.count < 0 || (apiKeyConfig.count - args?.dois?.length) < 0) {
```

Comment:
Useless condition here

```suggestion
      if ((apiKeyConfig.count - args?.dois?.length) < 0) {
```